docs/plugins/smartplaylist.rst

```diff
@@ -30,6 +30,26 @@ use spaces in the query, be sure to quote them (e.g., ``artist:"The Beatles"``).
 If you have existing files with the same names, you should back them up---they
 will be overwritten when the plugin runs.
 
+You also can generate a playlists to play music on your NAS or any server. Let
+assume that you host all your music at ``https://nas.domain.com/music``, that is
+mounted to your machine when you run beet as ``/mnt/music`` and you should login
+to it as user ``user`` with password ``secret``::
```

Comment:
Could we put this in a separate section, because it's separate from the basic instructions? Also, I'm not sure we need an example config with all the other settings, such as `playlists:`. Can we keep the example compact (if an example is necessary at all)?

Reply:
Let me remove this example because it is tricky one.